### PR TITLE
Update push notification for rewards distribution

### LINKED ISSUE #650: Update push notification for rewards distribution
### What would you like?
Change the text of the push notifications sent for rewards distributions

### Why is this needed?
We want a different text

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Changed the text for rewards push notifications and fetch them from a dedicated array meant for this.

### Other changes

Updated the node env to 12.

### Tested

Yes

### How others should test

See the text on the push notification you receive in the next distribution

### Related issues

- Fixes #650

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/notification-service/src/firebase.ts [hunk 1]
```diff
@@ -117,13 +114,23 @@ export function initializeDb() {
...
     (snapshot) => {
       const knownAddressesInfo: AddressToDisplayNameType = (snapshot && snapshot.val()) || {}
-      updateCeloRewardsSenderAddresses(knownAddressesInfo)
-      console.debug('Latest known addresses updated: ', celoRewardsSenders)
+      console.debug('Latest known addresses updated: ', knownAddressesInfo)
     },
     (errorObject: any) => {
       console.error('Known addresses data read failed:', errorObject.code)
     }
   )
```
Comment (jeanregisser) [inline packages/notification-service/src/firebase.ts:122]: Do we still need this listener?

File: packages/notification-service/locales/es.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
   "gold": "CELO",
   "paymentRequestedTitle": "Pago solicitado",
   "paymentRequestedBody": "Se ha solicitado pago de {{amount}} {{currency}}.",
-  "rewardReceivedTitle": "Recompensa recibida",
+  "rewardReceivedTitle": "Acabas de ganar más cUSD 📈",
+  "rewardReceivedBody": "¡Tu recompensa semanal ha llegado! Agrega cUSD para ganar aún más la próxima semana.",
```
Comment (jeanregisser) [inline packages/notification-service/locales/es.json:12]: We should not touch non US locales. It can creates conflicts in Crowdin PRs.

**Review by jeanregisser — COMMENTED** [review]
Great!

File: packages/notification-service/src/firebase.ts (hunk 1 above)
Comment (gnardini) [reply packages/notification-service/src/firebase.ts:122]: We don't, thanks!

**Review by jeanregisser — APPROVED** [review]
LGTM 👍

**Pull request merged**